functions/Get-DbaPrivilege.ps1

```diff
@@ -51,17 +51,19 @@ function Get-DbaPrivilege
     [PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential
   )
 
-BEGIN
+  BEGIN
   {
-    function Convert-SIDToUserName ([string] $SID ) {
-      $objSID = New-Object System.Security.Principal.SecurityIdentifier ("$SID") 
-      $objUser = $objSID.Translate( [System.Security.Principal.NTAccount]) 
-      $objUser.Value
+    $ResolveSID = @"
```

Comment:
I'm not sure, @ctrlbold . This is based on an example I found.
Surely repeating the whole block three times is not better?
When I have the time I can search for more concise methods, but this was the first that seems to work fine.
@niphlod @gbargsley can you confirm this resolves correctly?